s3cache/__init__.py

```diff
@@ -75,6 +75,11 @@ def __init__(self, _location, params):
         # sanitize location by removing leading and traling slashes
         self._options['LOCATION'] = self._location.strip('/')
 
+        if self._options.get('PICKLE_VERSION', None):
```

Comment:
That's minor, maybe if statement is redundant here:

```
self._pickle_protocol = self._options.get('PICKLE_VERSION', pickle.HIGHEST_PROTOCOL):
```

Reply:
I was trying to also ensure that PICKLE_VERSION is not set to None or an empty string.

Reply:
My thought was: in case bad configuration (None, '') is better to fail then silently pick up highest protocol.

However, I see your point and I'm good with it as well.

Reply:
Yeah, I think that makes sense too, so I can change it. Out of curiosity, I checked, and `pickle.dumps()` accepts a number or None (for highest) but not an empty string.
```
>>> pickle.dumps('hi', None)
b'\x80\x03X\x02\x00\x00\x00hiq\x00.'
>>> pickle.dumps('hi', '')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: an integer is required (got type str)
```